### DISCUSSION TITLE
Is there a specific reason why the docs discourage sqlite in production?

### DISCUSSION THREAD

**Discussion by tinco** [ory/keto#1077] [Q&A]
On this page it says a "production-grade" sql database is needed to run Keto in production:

https://www.ory.sh/docs/keto/guides/production

In the past year or two there's been a bit of a hype around running sqlite3 in production. I think because it's possible to run sqlite in multithreaded mode, and also because it's now easy to have multiple sqlite replicas read from a single sqlite master. I am not sure if I'm 100% on board, but I like the idea of using sqlite in a latency sensitive application such as Keto.

Are there any specific concerns with running Keto on sqlite beyond the obvious sensitivity to certain performance bottlenecks?

**Comment by vinckr** [comment]
I think @zepatrik is probably the best person to answer this. but Ory supports sqlite as first-class citizen so should be fine as long as your db can handle it.
